script/run-in-env.sh

```diff
@@ -9,9 +9,23 @@ if [ -s .python-version ]; then
 fi
 
 # other common virtualenvs
+my_path="."
+sed_cmd=`which sed`
+if [ "${sed_cmd}" == "" ]; then
+	if [ -f "/usr/bin/sed" ]; then
+		sed_cmd="/usr/bin/sed"
+	fi
+fi
+if [ "${sed_cmd}" != "" ]; then
+	if [ "$0" != "" ]; then
+		my_path=`echo $0 | ${sed_cmd} -e 's;\/[^/]*$;;'`
+		my_path="${my_path}/.."
+	fi
+fi
+
 for venv in venv .venv .; do
-    if [ -f $venv/bin/activate ]; then
-        . $venv/bin/activate
+    if [ -f ${my_path}/$venv/bin/activate ]; then
+        . ${my_path}/$venv/bin/activate
```

Comment:
Same here, quoting to prevent unexpected expansion.

Reply:
Put between quotes.